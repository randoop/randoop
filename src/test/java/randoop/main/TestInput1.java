package randoop.main;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInput1 {
  public static boolean debug = false;

  @Test
  public void test1() throws Throwable {
    if (debug) {
      System.out.format("%n%s%n", "ErrorTest0.test10");
    }
    org.apache.commons.lang3.math.Fraction fraction0 =
        org.apache.commons.lang3.math.Fraction.ONE_QUARTER;
    org.apache.commons.lang3.math.Fraction fraction1 =
        org.apache.commons.lang3.math.Fraction.TWO_QUARTERS;
    long long2 = fraction1.longValue();
    String str3 = fraction1.toProperString();
    org.apache.commons.lang3.math.Fraction fraction4 = fraction0.divideBy(fraction1);
    org.apache.commons.lang3.mutable.MutableByte mutableByte5 =
        new org.apache.commons.lang3.mutable.MutableByte((Number) fraction1);
    // byte byte6 = mutableByte5.getAndDecrement();

    // Checks the contract: compareTo-equals on fraction1 and fraction4
    org.junit.Assert.assertTrue(
        "Contract failed: compareTo-equals on fraction1 and fraction4",
        (fraction1.compareTo(fraction4) == 0) == fraction1.equals(fraction4));
  }
}
