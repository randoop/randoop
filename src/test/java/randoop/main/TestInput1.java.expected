package test;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;
import org.apache.commons.lang3.math.Fraction;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInput1Minimized {

    public static boolean debug = false;

    @Test
    public void test1() throws Throwable {
        Fraction fraction0 = Fraction.ONE_QUARTER;
        Fraction fraction1 = Fraction.TWO_QUARTERS;
        Fraction fraction4 = fraction0.divideBy(fraction1);
        // Checks the contract: compareTo-equals on fraction1 and fraction4
        org.junit.Assert.assertTrue("Contract failed: compareTo-equals on fraction1 and fraction4", (fraction1.compareTo(fraction4) == 0) == fraction1.equals(fraction4));
    }
}
